test/gen-matter-4.test.js

```diff
@@ -149,7 +149,7 @@ test(
     expect(ept).toContain('#define GENERATED_CLUSTER_COUNT 81')
     expect(ept).toContain('#define ZAP_FIXED_ENDPOINT_DATA_VERSION_COUNT 79')
     expect(ept).toContain('#define ATTRIBUTE_SINGLETONS_SIZE (37)')
-    expect(ept).toContain('#define ATTRIBUTE_MAX_SIZE (3998)')
+    expect(ept).toContain('#define ATTRIBUTE_MAX_SIZE (3988)')
```

Comment:
Why did the attribute size change due to your changes?

Reply:
see https://csamembers.slack.com/archives/C01B01GH64E/p1695906005160549